virttest/env_process.py

```diff
@@ -55,6 +53,12 @@
 from virttest.test_setup.networking import NetworkProxies
 
 
+# lazy imports for dependencies that are not needed in all modes of use
+from virttest._wrappers import lazy_import
+utils_libvirtd = lazy_import("utils_libvirtd")
+virsh = lazy_import("virsh")
```

Comment:
> @luckyh I followed your request but oddly ended up with `AttributeError: 'NoneType' object has no attribute 'loader'` so I will have to look a bit deeper in the suggested implementation and test it myself, it seems I have to supply different (complete?) namespace for the lazily imported modules.

@pevogam after checking the code changes, I guess that the issue was caused by intending on doing implicit relative imports. So would you please try with the following tweak to see if it could fix the issue?

```suggestion
utils_libvirtd = lazy_import("virttest.utils_libvirtd")
virsh = lazy_import("virttest.virsh")
```

Btw, we can apply such a check to get a better hint to the import issue you just saw, thanks!

```python
def lazy_import(name):
    spec = importlib.util.find_spec(name)
    if spec is None:
        raise ImportError(...)
    ...
```

Reply:
Ah indeed makes complete sense, pushed with the additional validation which is also a good idea.